packages/compiler-sfc/src/parse.ts

```diff
@@ -138,8 +133,7 @@ export function parse(
     pad = false,
     ignoreEmpty = true,
     compiler = CompilerDOM,
-    templateParseOptions = {},
-    parseExpressions = true,
+    templateParseOptions = { prefixIdentifiers: true },
```

Comment:
The default value will only be used when no `templateParseOptions` is passed, causing a regression for template import usage check.

Fixed in 3d6f01571